Vagrantfile

```diff
@@ -2,19 +2,19 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.define "ub1404" do |ub1404|
-    ub1404.vm.box = "metasploitable3-ub1404"
-    ub1404.vm.hostname = "metasploitable3-ub1404"
-    config.ssh.username = 'vagrant'
-    config.ssh.password = 'vagrant'
+#  config.vm.define "ub1404" do |ub1404|
```

Comment:
I am guessing you commented this out during testing.   Please revert this change unless there is some reason is should be removed.